modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewDXPKey/index.js

```diff
@@ -0,0 +1,318 @@
+										{productVersions?.map((version) => (
+											<ClaySelect.Option
+												key={version.label}
+												label={version.label}
+											/>
+										))}
+									</ClaySelect>
+
+									<ClayIcon
+										className="select-icon"
+										symbol="caret-bottom"
+									/>
+								</div>
+							</div>
+						</div>
+
+						<div className="mt-4 w-100">
+							<label htmlFor="basicInput">Key Type</label>
+
+							<div className="position-relative">
+								<ClaySelect
+									className="mr-2 pr-6 w-100"
+									onChange={({target}) => {
+										setSelectedKeyType(target.value);
+									}}
+									value={selectedKeyType}
+								>
+									{productKeyTypes &&
+										productKeyTypes[
+											returnSelectedVersionIndex
+										]?.map((keyType) => (
+											<ClaySelect.Option
+												key={keyType}
+												label={keyType}
+											/>
+										))}
+								</ClaySelect>
+
+								<ClayIcon
+									className="select-icon"
+									symbol="caret-bottom"
+								/>
+							</div>
+						</div>
+
+						<div>
+							<div className="mb-3 mt-4">
+								<h5>Subscription</h5>
+							</div>
+
+							<div>
+								{getSubscriptionTerms.map(
+									(subscriptionterm, index) => {
+										const wasSelected =
+											selected === subscriptionterm;
+										const displayAlertType = hasNotPermanentLicence ? (
+											<ClayAlert
+												className="px-4 py-3"
+												displayType="info"
+											>
+												<span className="text-paragraph">
+													Activation Key(s) will be
+													valid{' '}
+												</span>
+
+												<b className="text-paragraph">
+													{`${getCurrentEndDate(
+														subscriptionterm.startDate
+													)} `}
+													-
+													{` ${getCurrentEndDate(
+														subscriptionterm.endDate
+													)}`}
+												</b>
+											</ClayAlert>
+										) : (
+											<ClayAlert
+												className="px-4 py-3"
+												displayType="info"
+											>
+												<span className="text-paragraph">
+													Activation Key(s) will be
+													valid{' '}
+												</span>
+
+												<b className="text-paragraph">
+													indefinitely starting
+													{` ${getCurrentEndDate(
+														subscriptionterm.startDate
+													)}, `}
+												</b>
+
+												<span className="text-paragraph">
+													or until manually
+													deactivated.
+												</span>
+											</ClayAlert>
+										);
+
+										return (
+											<Radio
+												description={`Key activation available: ${subscriptionterm.provisionedCount} of ${subscriptionterm.quantity}`}
+												hasCustomAlert={
+													wasSelected &&
+													displayAlertType
+												}
+												isActivationKeyAvailable={
+													subscriptionterm.provisionedCount >
+													0
+														? true
+														: false
```

Comment:
Omit the true false return
```suggestion
													subscriptionterm.provisionedCount >
													0
													
```

Reply:
Done.